themes/src/main/resources/theme/keycloak/login/resources/css/login.css

```diff
@@ -574,6 +574,81 @@ ul#kc-totp-supported-apps {
     width: 40%;
 }
 
+/* Internet Explorer 11 compatibility workaround for select-authenticator screen */
```

Comment:
I'm afraid I have to bug you a bit more with IE. It seems there's some overflow when the text is longer.
![Capture](https://user-images.githubusercontent.com/13906855/98124882-09113000-1eb4-11eb-8c3c-859191ddd17c.PNG)

Reply:
Fixed by adding:
https://github.com/Pepo48/keycloak/blob/4a2849b1467b8567a48328a776acef004b568e6e/themes/src/main/resources/theme/keycloak/login/resources/css/login.css#L605-L608

It just seems to be another IE specific bug/feature, where it needs the explicit request[1].
[1] https://stackoverflow.com/a/35113633